crates/bws/src/command/project.rs

```diff
@@ -0,0 +1,119 @@
+use std::process;
+
+use bitwarden::{
+    secrets_manager::projects::{
+        ProjectCreateRequest, ProjectGetRequest, ProjectPutRequest, ProjectsDeleteRequest,
+        ProjectsListRequest,
+    },
+    Client,
+};
+use color_eyre::eyre::Result;
+use uuid::Uuid;
+
+use super::OutputSettings;
+use crate::render::serialize_response;
+
+pub(crate) async fn list(
+    mut client: Client,
+    organization_id: Uuid,
+    output_settings: OutputSettings,
+) -> Result<()> {
+    let projects = client
+        .projects()
+        .list(&ProjectsListRequest { organization_id })
+        .await?
+        .data;
+    serialize_response(projects, output_settings.output, output_settings.color);
```

Comment:
**suggestion:** We can change `serialize_response` to accept `OutputSettings` which would be slightly cleaner.

Reply:
Updated in: https://github.com/bitwarden/sdk/pull/809/commits/826ba67ed9baa5103f575fdbc7f614846cd3ea54